### ISSUE TITLE
Request executable. 

### ISSUE THREAD

**Issue by Leomist** [airbreather/HQMEditor#1]
I have attempted to build this project; however I have wasted so much time just trying to get it to build. 

Would you have the built project still? 

I'm working on a old version of HQM for Minecraft 1.6.4

Hopefully. 
Thanks. Leo.

**Comment by airbreather** [comment]
No, sorry, and it's been way too long since I last touched this project for me to be able to figure it out again :disappointed:
